src/ci/docker/scripts/crosstool-ng-git.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -ex
+
+git clone --depth=1 https://github.com/crosstool-ng/crosstool-ng
```

Comment:
This should be pinned to a fixed commit, to make sure we don't pick up a broken commit.